share/completions/modinfo.fish

```diff
@@ -0,0 +1,16 @@
+function __fish_print_modules
```

Comment:
This is the same thing as in the modprobe completions. It should be split out into its own file in share/functions.